### ISSUE TITLE
不支持地理查询

### ISSUE THREAD

**Issue by jianbingguozi** [AntonShuvaev/elasticsearch4idea#55]
不支持地理查询语句 like this
```rest
GET /demo/_search
{
  "from": 0,
  "size": 10,
  "sort": [
    {
      "_geo_distance": {
        "location": {
          "lat": 30,
          "lon": 121
        },
        "order": "desc",
        "unit": "km"
      }
    }
  ],
  "query": {
    "match_all": {}
  }
}
```
execute 之后会格式化为以下错误形式
```rest
GET /demo/_search
{
  "from": 0,
  "size": 10,
  "sort": [
    {
      "_geo_distance": {
        "order": "desc"
      }
    }
  ],
  "query": {
    "match_all": {}
  }
}

```

version:1.7.0-2021.2

**Comment by AntonShuvaev** [comment]
Thanks for reporting this issue. Will be fixed in the next update.

**Comment by AntonShuvaev** [comment]
Fixed in 1.7.1
